compiler/next/test/parsing/testParseImport.cpp

```diff
@@ -0,0 +1,210 @@
+      "/*c1*/\n"
+      "public import /*c2*/ A as X, /*c3*/ B.SM1 as Y, /*c4*/ C as Z;\n"
+      "/*c5*/\n");
+  assert(parseResult.errors.size() == 0);
+  assert(parseResult.topLevelExpressions.size() == 1);
+  assert(parseResult.topLevelExpressions[0]->isModule());
+  auto mod = parseResult.topLevelExpressions[0]->toModule();
+  assert(mod->numStmts() == 3);
+  assert(mod->stmt(0)->isComment());
+  assert(mod->stmt(1)->isImport());
+  assert(mod->stmt(2)->isComment());
+  const Import* imp = mod->stmt(1)->toImport();
+  assert(imp->visibility() == Decl::PUBLIC);
+  assert(imp->numVisibilityClauses() == 3);
+  // Inspect the first visibility clause.
+  {
+    const VisibilityClause* visClause = imp->visibilityClause(0);
+    assert(visClause->symbol()->isAs());
+    const As* as = visClause->symbol()->toAs();
+    assert(as->symbol()->isIdentifier());
+    assert(as->symbol()->toIdentifier()->name() == "A");
+    assert(as->rename()->isIdentifier());
+    assert(as->rename()->toIdentifier()->name() == "X");
+    assert(visClause->limitationKind() == VisibilityClause::NONE);
+    assert(visClause->numLimitations() == 0);
+  }
+  // Inspect the second visibility clause.
+  {
+    const VisibilityClause* visClause = imp->visibilityClause(1);
+    assert(visClause->symbol()->isAs());
+    const As* as = visClause->symbol()->toAs();
+    assert(as->symbol()->isDot());
+    const Dot* dot = as->symbol()->toDot();
+    assert(dot->numActuals() == 0);
+    assert(dot->calledExpression()->isIdentifier());
+    assert(dot->calledExpression()->toIdentifier()->name() == "B");
+    assert(dot->field() == "SM1");
+    assert(as->rename()->isIdentifier());
+    assert(as->rename()->toIdentifier()->name() == "Y");
+    assert(visClause->limitationKind() == VisibilityClause::NONE);
+    assert(visClause->numLimitations() == 0);
+  }
+  // Inspect the third visibility clause.
+  {
+    const VisibilityClause* visClause = imp->visibilityClause(2);
+    assert(visClause->symbol()->isAs());
+    const As* as = visClause->symbol()->toAs();
+    assert(as->symbol()->isIdentifier());
+    assert(as->symbol()->toIdentifier()->name() == "C");
+    assert(as->rename()->isIdentifier());
+    assert(as->rename()->toIdentifier()->name() == "Z");
+    assert(visClause->limitationKind() == VisibilityClause::NONE);
+    assert(visClause->numLimitations() == 0);
+  }
+  // Make sure the 'visibilityClauses' iterator works as expected.
+  {
+    int count = 0;
+    for (const auto visClause : imp->visibilityClauses()) {
+      assert(visClause->tag() == asttags::VisibilityClause);
+      assert(visClause->isVisibilityClause());
+      assert(visClause == imp->visibilityClause(count));
+      count++;
+    }
+    assert(count == imp->numVisibilityClauses());
+  }
+}
+
+static void test2(Parser* parser) {
+  auto parseResult = parser->parseString("test2.chpl",
+      "/*c1*/\n"
+      "private import /*c2*/ A as X, B.{Y, Z};\n"
```

Comment:
It'd be good to have a test with:
- Just a single symbol inside the curly braces
- Renaming one of the symbols inside the curly braces